tests/test_framework/utils.py

```diff
@@ -338,26 +340,34 @@ def wrapper(*args, **kwargs):
     def call(self, method, payload=[]):
         self.logger.debug("Calling %s with payload %r", method, payload)
 
-        # FIXME: we open a new socket for every readobj call...
-        sock = UnixSocket(self.socket_path)
-        msg = json.dumps(
-            {
-                "jsonrpc": "2.0",
-                "id": 0,
-                "method": method,
-                "params": payload,
-            }
-        )
-        sock.sock.send(msg.encode())
-        this_id = self.next_id
-        resp = self._readobj(sock)
-
-        self.logger.debug("Received response for %s call: %r", method, resp)
-        if "id" in resp and resp["id"] != this_id:
-            raise ValueError(
-                "Malformed response, id is not {}: {}.".format(this_id, resp)
+        if platform.uname()[0] == "Windows":
```

Comment:
nit: you can just use `system()`: https://docs.python.org/3/library/platform.html#platform.system